daemon/main.c

```diff
@@ -344,7 +348,20 @@ static const char *verify_required_directory(const char *dir) {
     return dir;
 }
 
-void log_init(void) {
+#ifdef ENABLE_HTTPS
+static void security_init(){
+    char filename[FILENAME_MAX + 1];
+    snprintfz(filename, FILENAME_MAX, "%s/key.pem",netdata_configured_user_config_dir);
+    security_key    = config_get(CONFIG_SECTION_WEB, "ssl key",  filename);
+
+    snprintfz(filename, FILENAME_MAX, "%s/cert.pem",netdata_configured_user_config_dir);
+    security_cert    = config_get(CONFIG_SECTION_WEB, "ssl certificate",  filename);
```

Comment:
I see that `key.pem` and `cert.pem` names are hardcoded. Not a security specialist, but i think there are no mandatory rule for key to have a name `key` and for cert to have a name `cert`. Not a security specialist tho.

Neither dir and name shouldn;t be hardcoded imo.

 @Ferroin what do you think?

Reply:
Due the fact we need to have default values, these are suggestion for the default key and certificate, this happens because we cannot let it as NULL.

Reply:
oh, these are defaults, ok.

Reply:
So by default we expecte both `cert.pem` and `key.pem` to be in the `/opt/netdata/etc/netdata` ? 

>  netdata_configured_user_config_dir  = config_get(CONFIG_SECTION_GLOBAL, "config directory",       netdata_configured_user_config_dir);

> # config directory = /opt/netdata/etc/netdata

Reply:
Yes this is expected, I think it is the better place, another possibility would be the netdata home directory.

Reply:
@Ferroin i am really interested in your opinion oh that

Reply:
why not an SSL directory within etc/netdata ?

Reply:
No problem, we can do this. I think this will organize better the files.